flink-libraries/flink-table/src/test/scala/org/apache/flink/table/api/scala/stream/sql/SqlITCase.scala

```diff
@@ -317,4 +317,92 @@ class SqlITCase extends StreamingWithStateTestBase {
     result.addSink(new StreamITCase.StringSink)
     env.execute()
   }
+  
+  
+  @Test
+  def testAvgSumAggregatationPartition(): Unit = {
```

Comment:
These test cannot correctly verify processing time OVER windows.
We need to add tests that allow us to control processing time.
Have a look at `WindowOperatorTest.testProcessingTimeSlidingWindows()` which uses the `KeyedOneInputStreamOperatorTestHarness` to control the processing time (that's the reason why we do not call `System.getMillis()`).

I think the required dependencies are also added in PR #3386.